.github/workflows/SDK-Suppressions-Label.yaml

```diff
@@ -0,0 +1,99 @@
+name: Manage SDK Breakingchange Suppressions Labels Workflow
+
+on:
+  pull_request:
+    branches:
+      - jacktn/suppression-action 
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+  workflow_dispatch:
+  
+jobs:
+  process-sdk-suppressions-labels:
+    name: breakingchange suppressions label Workflow
```

Comment:
Can you use a short name like SBC-SL? It stands for `SdkBreakingChange-SuppressionLabel`. You may add the full name as comment.

Reply:
renamed

Reply:
@JackTn can you use short name: `SBC-SL` to save space while rendering the name in the result UI?